Google.PowerShell/Common/ReportCmdletResults.cs

```diff
@@ -174,32 +180,33 @@ public GoogleAnalyticsCmdletReporter(string clientID, AnalyticsEventCategory ana
             }
         }
 
-        public void ReportSuccess(string cmdletName, string parameterSet)
+        public void ReportSuccess(string cmdletName, string parameterSet, string projectNumber)
         {
-            Report(cmdletName, parameterSet, null);
+            Report(cmdletName, parameterSet, projectNumber, null);
         }
 
-        public void ReportFailure(string cmdletName, string parameterSet, int errorCode)
+        public void ReportFailure(string cmdletName, string parameterSet, string projectNumber, int errorCode)
         {
-            Report(cmdletName, parameterSet, errorCode);
+            Report(cmdletName, parameterSet, projectNumber, errorCode);
         }
 
-        private void Report(string cmdletName, string parameterSet, int? errorCode)
+        private void Report(string cmdletName, string parameterSet, string projectNumber, int? errorCode)
         {
             // The event type will be CmdletInvocation or ProviderInvocation. The specific cmdlet and parameter set
             // used are encoded in the event's metadata.
             AnalyticsEvent eventData = new AnalyticsEvent(
                 _category.ToString(),
                 "cmdletName", cmdletName,
                 "parameterSet", (parameterSet ?? "null"),
+                "projectNumber", (projectNumber ?? "null"),
                 "errorCode", (errorCode.HasValue ? errorCode.ToString() : "null"));
 
             s_reporter.Value.ReportEvent(
                 source: "virtual.powershell",
                 eventType: "powershell",
                 eventName: eventData.Name,
                 userLoggedIn: true,
-                projectNumber: null,
+                projectNumber: projectNumber,
```

Comment:
Something seems wrong here. Projects can be identified both by name ("bobs-taco-truck") and also by a number (29576092874). I assume we need to do some sort of conversion between the two?

Alternatively, should we just return `projectName.GetHashCode` instead? i.e. an anonymous variant of the project name?

@ivannaranjo , can you chime in here? We aren't reporting project info in our telemetry. What is the right way to wire this in?